.changeset/odd-buckets-attend.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lit/reactive-element': patch
```

Comment:
reactive-element patch should also patch `lit`:

```suggestion
'@lit/reactive-element': patch
'lit': patch
```

Reply:
I think we should also do `lit-element` as it depends on `@lit/reactive-element`

Reply:
Updated suggestion. Thank you!

Reply:
done